lib/Doctrine/Common/Cache/FileCache.php

```diff
@@ -164,7 +164,7 @@ protected function doDelete($id)
     {
         $filename = $this->getFilename($id);
 
-        return @unlink($filename) || ! file_exists($filename);
```

Comment:
True we could add another check after unlink. That way the functions response will be the same but we will still get an error in the event that something needs to be debugged.